core/src/main/scala/org/apache/spark/internal/io/FileCommitProtocol.scala

```diff
@@ -174,3 +233,12 @@ object FileCommitProtocol extends Logging {
     new Path(path, ".spark-staging-" + jobId)
   }
 }
+
+/**
+ * The specification for Spark output file name.
+ * This is used by [[FileCommitProtocol]] to create full path of file.
+ *
+ * @param prefix Prefix of file.
+ * @param ext Extension of file.
+ */
+final case class FileNameSpec(prefix: String, ext: String)
```

Comment:
shall we rename `ext` to `suffix`? it reality it includes more than extension, such as file count.

Reply:
@cloud-fan - agree, updated.